src/Api/Endpoints/Events.cs

```diff
@@ -0,0 +1,47 @@
+using System.ComponentModel.DataAnnotations;
+using ApiHelpers;
+using MiniValidation;
+using Passwordless.Api.Authorization;
+using Passwordless.Service.AuditLog.Loggers;
+using Passwordless.Service.AuditLog.Mappings;
+using Passwordless.Service.Features;
+
+namespace Passwordless.Server.Endpoints;
+
+public static class AuditLog
+{
+    public static void MapAuditLogEndpoints(this WebApplication app)
+    {
+        app.MapGet("events", GetAuditLogEvents)
+            .RequireSecretKey()
+            .RequireCors("default");
+    }
+
+    private static async Task<IResult> GetAuditLogEvents(
+        HttpRequest request,
+        IAuditLogStorage storage,
+        IFeatureContextProvider provider,
+        CancellationToken cancellationToken,
+        int pageNumber,
+        [MaxLength(1000)] int numberOfResults = 100)
```

Comment:
@jrmccannon Not sure, but wouldn't MaxLength be for a length, such as string length and not maxmium numeric value?

Reply:
Good catch. It took a bit of tinkering, but I got the validation working correctly. I was able to use `Range(1,1000)`.